mlserver/utils.py

```diff
@@ -1,11 +1,16 @@
 import os
+import logging
 
 from typing import List
 
 from .settings import ModelSettings
 from .errors import InvalidModelURI
 
 
+# In order for logging to be captured we need uvicorn logger
+logger = logging.getLogger("uvicorn.error")
```

Comment:
Interesting! Do you know why is this the case? Are other loggers getting hidden?